src/kuzya.cpp

```diff
@@ -659,6 +659,11 @@ void Kuzya::slotSetFileSuffix(QStringList filter)
     }
 }
 
+void Kuzya::slotSetFileSuffix(QString suffix)
+{   //this methods explicitly converts QString to QstrinList. Using by singals/slot connection
+    slotSetFileSuffix(static_cast<QStringList>(suffix));
```

Comment:
Please, promise me, never do it again) How you could convert in such way? 
If you need QStringList - you could just do this: 
`slotSetFileSuffix(QStringList() << suffix)`

P. S. Did you test it after you changes? Always test if it works. Cuz, I believe it should crash with static cast.